### PR TITLE
[Docs] `overview.md` and `migration.md`

### PR DESCRIPTION
**Pull request by DeemoZen**
Improved from master branch PR#780, 'migration.md' modified according to review, 'overview.md' adjusted the space between English and Chinese. For details, please see https://github.com/open-mmlab/mmtracking/pull/780/files

### CODE REVIEW

**Comment by JingweiZhang12** [comment]
@DeemoZen Please fix the lint.

File: docs_deprecated/zh_cn/migration.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+#  MMTracking 版本迁移
+
+与 0.xx 版本的 MMTracking 相比, 最新的 1.xx 版本有以下重要修改。
+
+## 整体架构
+
+MMTracking 0.xx 版本中的`core`，在 1.xx 的新版中被划分为了`engine`、 `evaluation`、 `structures`、 `visualization` 和 `model/task_moduls` 模块。详细信息可见[用户指南](../../docs/en/user_guides)。
```
Comment (JingweiZhang12) [inline docs_deprecated/zh_cn/migration.md:7]: Black space before `core`

File: docs_deprecated/zh_cn/migration.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+## 模型
+
+### 数据预处理
+
+MMTracking 的 1.xx 版本增加了 [TrackDataPreprocessor](../../mmtrack/models/data_preprocessors/data_preprocessor.py)。由数据 pipeline 输出的数据经该模块的转换后，再被送入到模型中。
+
+### 训练
+
+在训练阶段的正向传播过程中，Models 及 Heads 的每一类结果都是调用`loss`函数来执行。模型中的`loss`函数包含一个值为`Tensor`的字典和一个元素为 `TrackDataSample`的列表。
```
Comment (JingweiZhang12) [inline docs_deprecated/zh_cn/migration.md:129]: Black space around `loss` and `Tensor`
在训练阶段的正向传播过程中，Models 和 Heads 类会通过 `loss` 函数来计算 loss。`loss` 函数的输入包括一个值为`Tensor`的字典和一个元素为 `TrackDataSample`的列表。

File: docs_deprecated/zh_cn/migration.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+### 测试
+
+在测试阶段的正向传播过程中，Models 及 Heads 的每一类结果都是调用`predict`函数来执行。模型中的`predict`函数包含一个值为`Tensor`的字典和一个元素为 `TrackDataSample`的列表。
```
Comment (JingweiZhang12) [inline docs_deprecated/zh_cn/migration.md:133]: Black space around `predict` and 'Tensor' and `TrackDataSample`。
在测试阶段的正向传播过程中，Models 及 Heads 会通过调用 `predict` 函数来得到预测结果。`predict`函数的输入包括一个值为`Tensor`的字典和一个元素为 `TrackDataSample`的列表

File: docs_deprecated/zh_cn/migration.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+MMTracking 的 1.xx 版本新增了两个数据结构: [TrackDataSample](../../mmtrack/structures/track_data_sample.py) 和 [ReIDDataSample](../../mmtrack/structures/reid_data_sample.py)。这些数据结构封装了一张图片或序列的真值标签及预测结果，并将其用作为不同组件的接口。
+
+### Dataset 类
+
+MMTracking 的 1.xx 版本新增了两个继承自 MMEngine 的 Dataset 基类：`BaseSOTDataset` 和 `BaseVideoDataset`。前者仅用于单目标跟踪（SOT），后者可用于其余所有任务。
+
+### Data Pipeline
+
+1. MMTracking 旧版中的大部分图像序列变换，在新版本中被重构。具体来说，我们使用`TransformBroadcaster` 来封装单个图像的变换。
```
Comment (JingweiZhang12) [inline docs_deprecated/zh_cn/migration.md:147]: Please check the black space.

File: docs_deprecated/zh_cn/migration.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+
+   我们仍旧保留了某些图像序列的变换，例如`SeqCropLikeStark`，因为`TransformBroadcaster`不支持分别为序列中的每个图像设置不同的参数。
```
Comment (JingweiZhang12) [inline docs_deprecated/zh_cn/migration.md:149]: Please check the black space.

File: docs_deprecated/zh_cn/migration.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+
+2. 我们将旧版的`VideoCollect`, `ConcatSameTypeFrames` 和 `SeqDefaultFormatBundle`进行了封装 ，在新版中使用`PackTrackInputs`。
```
Comment (JingweiZhang12) [inline docs_deprecated/zh_cn/migration.md:151]: Please check the black space.

File: docs_deprecated/zh_cn/migration.md [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+
+3. 旧版本 pipeline 中的 normalizaion 被移除，其操作移至模型的正向传播过程。
+
+### 数据采样
+
+MMTracking 的 1.xx 版本新增了 `DATA_SAMPLERS` 注册器。您可以在配置中自定义不同的数据集采样器，详细信息参见[此处](../../mmtrack/datasets/samplers)。
+
+## 评估
+
+在旧版本的 MMTarcking 中，评估过程使用的是 Dataset 类。在 MMTracking 的 1.xx 版本中，我们新增了`METRICS`器，所有的评估都在 metric 类中实现，它们都注册在 `METRICS`中，详细信息参见[此处](../../mmtrack/evaluation/metrics)。
```
Comment (JingweiZhang12) [inline docs_deprecated/zh_cn/migration.md:161]: Please check the black space.  我们新增了`METRICS`器 > 我们新增了 `METRICS` 注册器

File: docs_deprecated/zh_cn/migration.md [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+
+## 可视化
+
+新增了`TrackLocalVisualizer` 和 `DetLocalVisualizer`，它们被注册进`VISUALIZER`注册器中。相较于旧版，新版支持了图像和特征图的可视化，详细信息参见[此处](../../mmtrack/visualization/local_visualizer.py)。
```
Comment (JingweiZhang12) [inline docs_deprecated/zh_cn/migration.md:165]: Please check the black space.

File: docs_deprecated/zh_cn/migration.md [hunk 9]
```diff
@@ -0,0 +1,169 @@
...
+
+## 引擎
+
+MMTracking 的 1.xx 版本重构了`runner`、`hook`、`logging`和`optimizer`，详细信息可见于 MMEngine。
```
Comment (JingweiZhang12) [inline docs_deprecated/zh_cn/migration.md:169]: Please check the black space.

**Review by JingweiZhang12 — APPROVED** [review]

**Review by GT9505 — COMMENTED** [review]
better to rebase dev-1.x branch or open a new pr in order to remove duplicated changes.

**Comment by JingweiZhang12** [comment]
@DeemoZen Please add the two docs under `docs/zh_cn`.

**Comment by JingweiZhang12** [comment]
@DeemoZen  Please remove the modification under `docs_deprecated`

**Review by GT9505 — APPROVED** [review]

**Pull request merged**